src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationScopeParsingTests.cs

```diff
@@ -2713,13 +2752,16 @@ public void Local_12()
                 {
                     N(SyntaxKind.LocalDeclarationStatement);
                     {
-                        N(SyntaxKind.ScopedKeyword);
                         N(SyntaxKind.ScopedKeyword);
```

Comment:
>N(SyntaxKind.ScopedKeyword); [](http://example.com/codeflow?start=24&length=28)

Why not `ScopedType` with nested `ScopedType`?

Reply:
> Why not ScopedType with nested ScopedType?

The way I read the grammar, `scoped` is not part of the type syntax. It is a token that can precede a type syntax in some places, therefore, there is no such thing like a ```ScopedType``` nested into a ```ScopedSype. Even though types that we use to represent the syntax allow this kind of nesting. The same story is for ```RefType```